compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -1083,10 +1083,11 @@ OMR::X86::CodeGenerator::supportsNonHelper(TR::SymbolReferenceTable::CommonNonhe
       case TR::SymbolReferenceTable::atomicAddSymbol:
       case TR::SymbolReferenceTable::atomicFetchAndAddSymbol:
       case TR::SymbolReferenceTable::atomicSwapSymbol:
-         {
          result = true;
          break;
-         }
+      default:
+         result = false;
```

Comment:
Do you need to assign it to `false` because that's what it is initialized to?  I'd say either just `break` or remove the initialization to `false` from L1079.

Reply:
No need for both, I removed the initialization at L1079.